Shared/Samples/Configure basemap style parameters/ConfigureBasemapStyleParametersView.swift

```diff
@@ -20,70 +20,56 @@ struct ConfigureBasemapStyleParametersView: View {
     @StateObject private var model = Model()
     
     /// The selected basemap style language strategy.
-    @State private var selectedLanguage: BasemapStyleLanguage = .global
-    
-    /// The selected locale.
-    @State private var selectedLocale: Locale = .current
+    @State private var selectedLanguage: BasemapStyleLanguage = .strategic(.global)
     
     var body: some View {
         MapView(map: model.map)
             .toolbar {
                 ToolbarItemGroup(placement: .bottomBar) {
                     Spacer()
                     languageMenu
-                        .onChange(of: selectedLanguage) { newLanguage in
-                            model.setBasemapLanguage(newLanguage)
-                        }
                     Spacer()
                 }
             }
     }
     
     private var languageMenu: some View {
         Menu("Language Settings") {
-            Section("Language Strategy") {
-                // A picker for specific languages.
-                Menu("Specific Language") {
-                    Picker(selectedLanguage.label, selection: $selectedLocale) {
-                        ForEach(model.languages, id: \.1) { label, code in
-                            Text(label).tag(Locale(identifier: code))
-                        }
-                    }
-                    .onChange(of: selectedLocale) { newLocale in
-                        selectedLanguage = .specific(newLocale)
-                    }
-                }
-                
-                // A series of buttons for general language strategies.
+            Menu("Strategic") {
                 ForEach(
-                    [
-                        // Use the default language setting for the basemap style.
-                        BasemapStyleLanguage.default,
-                        // Use the global language (English) for basemap labels.
-                        BasemapStyleLanguage.global,
-                        // Uses country-local language for basemap labels.
-                        BasemapStyleLanguage.local,
-                        // Use the system locale language for basemap labels.
-                        BasemapStyleLanguage.applicationLocale
-                    ],
+                    [BasemapStyleLanguageStrategy.default, .global, .local, .applicationLocale],
```

Comment:
Can you retain the comments so the audience can see the difference between the strategies more easily? Thank you.

Reply:
Aren't the comments simply repeating what is in the documentation?

Reply:
Yes, I thought adding some comment would help the reader to spot the key place to look at in the code. OK not to add.